pkg/execution/history/lifecycle.go

```diff
@@ -486,6 +487,126 @@ func (l lifecycle) OnWaitForEventResumed(
 	}
 }
 
+// OnInvokeFunction is called when a function is invoked from a step.
+func (l lifecycle) OnInvokeFunction(
+	ctx context.Context,
+	id state.Identifier,
+	item queue.Item,
+	op state.GeneratorOpcode,
+	eventID ulid.ULID,
+	corrID string,
+) {
+	logger.From(ctx).Debug().Interface("id", id).Msg("OnInvokeFunction")
+
+	groupID, err := toUUID(item.GroupID)
+	if err != nil {
+		l.log.Error(
+			"error parsing group ID",
+			"error", err,
+			"group_id", item.GroupID,
+			"run_id", id.RunID.String(),
+		)
+	}
+
+	fnID := ""
+	expiry := time.Time{}
+
+	opts, err := op.InvokeFunctionOpts()
+	if err != nil {
+		l.log.Error("error parsing invoke function options", "error", err)
+	}
+
+	if opts != nil {
+		fnID = opts.FunctionID
+		optsExp, err := opts.Expires()
+		if err != nil {
+			l.log.Error("error parsing invoke function options expiry", "error", err)
+		} else {
+			expiry = optsExp
+		}
+	} else {
+		l.log.Error("invoke function options are nil")
+	}
+
+	h := History{
+		ID:              ulid.MustNew(ulid.Now(), rand.Reader),
```

Comment:
Nit: Can you alphabetize the fields? It makes it easier to see if we're missing one, which is tough when there are tons of them. I need to put up a PR to alphabetize the preexisting ones too

Reply:
@goodoldneon Is there something that does that for me? 😬

Reply:
I use vim's `:sort` but VS Code has an equivalent!
<img width="287" alt="image" src="https://github.com/inngest/inngest/assets/20100586/b1203b2b-3a69-4072-b35f-1865128f3cee">